array-methods.js

```diff
@@ -81,18 +159,30 @@ var higherStateSums = null;
     Ohio
     Georgia
     Delaware
-
+ 
   Check if all of these states have a sum of account values
   greater than 2,550,000
-
+ 
   if true set `areStatesInHigherStateSum` to `true`
   otherwise set it to `false`
  */
-var areStatesInHigherStateSum = null;
+var areStatesInHigherStateSum = Object.entries(stateSums).filter(function (element) {
+  if (['WI', 'IL', 'WY', 'GA', 'DE', 'OH'].includes(element[0])) {
+    return true;
+  } else {
+    return false;
+  }
+}).every(function (element) {
+  if (element[1] > 2550000) {
```

Comment:
You could just return the result of the if conditional here as well since `.every` is just looking for a true/false value being returned.